heart/heartbeat.py

```diff
@@ -77,17 +93,17 @@ def calc_ib_version(w,x,y,a,z):
 		# Default priority values
 		PRIORITY_DEFAULT = p.get("default_priority", PRIORITY_DEFAULT)
 		PRIORITY_OFFICIAL = p.get("official_priority", PRIORITY_OFFICIAL)
-		# Cleanup
-		del p
+	# Cleanup, since we are in global scope
+	del p, k, v, fp
+
 except IOError:
 	print "Could not read servers.json - skipping"
 except AttributeError:
 	print "Error in servers.json - skipping"
 
-def stripnul(s):
-	idx = s.find("\x00")
-	return (s if idx == -1 else s[:idx])
-
+"""
+# this function does some weird stuff, either me or the author has no idea what he is doing
+# until stuff breaks, it has been replaced by str.replace
```

Comment:
We'd need to actually test it for that, and for that we need greaser. Yes, it would be easier to just use his code, but It's not like this pull is time critical. Using python's code for this would also Take care of the n bit variants too.